src/openforms/submissions/tests/form_logic/test_modify_components.py

```diff
@@ -958,6 +958,54 @@ def test_dirty_data_only_diff_data_returned(self):
             {"changingKey": "changed"},
         )
 
+    def test_select_boxes_trigger(self):
+        form = FormFactory.create()
+        step = FormStepFactory.create(
+            form=form,
+            form_definition__configuration={
+                "components": [
+                    {
+                        "type": "selectboxes",
+                        "key": "currentPets",
+                        "values": [
+                            {"label": "Cat", "value": "cat"},
+                            {"label": "Dog", "value": "dog"},
+                            {"label": "Fish", "value": "fish"},
+                        ],
+                    }
+                ]
+            },
+        )
+        FormLogicFactory.create(
+            form=form,
+            json_logic_trigger={
+                "==": [
+                    {"var": "currentPets.cat"},
+                    True,
+                ]
```

Comment:
I don't think we can actually specify boolean values at the moment in the logic builder, they get interpreted as a string. Not sure if json-logic package handles this conversion?

Reply:
Because of this: https://github.com/open-formulieren/open-forms/blob/fix/891-logic-selectboxes/src/openforms/js/components/admin/form_design/logic/Trigger.js#L276 the value is automatically set to a bool in the admin. It's a bit hackish maybe :sweat_smile:

Reply:
Hrm, so that makes it not possible to attach logic if the value is `false` instead of `true`?

Reply:
I thought that in that case you can use the `!=` operator